pgxn/neon/walproposer_pg.c

```diff
@@ -500,8 +552,8 @@ walprop_register_bgworker(void)
 {
 	BackgroundWorker bgw;
 
-	/* If no wal acceptors are specified, don't start the background worker. */
-	if (*wal_acceptors_list == '\0')
+	/* If no safekeepers are specified, don't start the background worker. */
+	if (*safekeeper_connstrings == '\0')
```

Comment:
Is it another place where we need to look at both the values for awhile, for compat?